### PR TITLE
PREAPPS-7350: Remove retention policy setting 

### PR DESCRIPTION
**Pull request by lavegupta**

### CODE REVIEW

**Comment by silentsakky** [comment]
There is conflict in zimbra-attrs.xml file
also we need to backport changes to ZimbraOS/zm-mailbox also

**Comment by lavegupta** [comment]
> There is conflict in zimbra-attrs.xml file also we need to backport changes to ZimbraOS/zm-mailbox also

Resolved and backport changes
https://github.com/ZimbraOS/zm-mailbox/pull/577/files

**Review by silentsakky — APPROVED** [review]

File: store/conf/attrs/zimbra-attrs.xml [hunk 1]
```diff
@@ -10379,4 +10379,11 @@ TODO: delete them permanently from here
   <defaultCOSValue>TRUE</defaultCOSValue>
   <desc>To restrict/allow the use of username in the password when user reset or change their password to achieve greater password security.</desc>
 </attr>
+
+<attr id="4101" name="zimbraFeatureRetentionPolicyEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="10.1.0">
+  <defaultCOSValue>TRUE</defaultCOSValue>
+  <defaultExternalCOSValue>FALSE</defaultExternalCOSValue>
+  <desc>Whether retention policy feature should be enabled for this account or cos</desc>
```
Comment (k-kato) [inline store/conf/attrs/zimbra-attrs.xml]:
```suggestion
  <desc>Whether retention policy feature is enabled</desc>
```

**Review by miteshsavani810 — APPROVED** [review]

File: store/conf/attrs/zimbra-attrs.xml (hunk 1 above)
Comment (silentsakky) [reply store/conf/attrs/zimbra-attrs.xml]: fixed

**Review by silentsakky — APPROVED** [review]

**Pull request merged**